### PR TITLE
Fixed bug #673 by making cell textLabel's number of lines as 0

### PR DESCRIPTION
**Pull request by danishjafri88**
## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-ios/issues) for my issue if one did not already exist.
- [x] My patch or PR title has a standard commit message that looks like `Fix #123: This fixes the shattered coffee cup!` (or `No Bug: <message>` if no relevant ticket)
- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`
- [x] New files have MPL-2.0 license header.

## Reviewer Checklist:

- [x] PR is linked to an issue via [Zenhub](https://www.zenhub.com/extension).
- [x] Issues are assigned to at least one epic.
- [x] Issues include necessary QA labels:
  - [x] `QA/(Yes|No)`
  - [x] `release-notes/(include|exclude)`
  - [x] `bug` / `enhancement`
- [x] Necessary security reviews have taken place.
- [x] Adequate test coverage exists to prevent regressions.
- [x] Adaquate test plan exists for QA to validate (if applicable)

### CODE REVIEW

File: Client/Frontend/Settings/SettingsViewController.swift [hunk 1]
```diff
@@ -310,27 +313,63 @@ class SettingsViewController: TableViewController {
...
                     alert.addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: nil))
                     self.present(alert, animated: true, completion: nil)
-                }, cellClass: ButtonCell.self)
+                }, cellClass: MultilineButtonCell.self)
             ]
         )
     }()
 }
+
+class MultilineButtonCell: ButtonCell {
```
Comment (kylehickinson) [inline Client/Frontend/Settings/SettingsViewController.swift]: These classes should all be `private`

**Review by kylehickinson — CHANGES_REQUESTED** [review]

File: Client/Frontend/Settings/SettingsViewController.swift (hunk 1 above)
Comment (danishjafri88) [reply Client/Frontend/Settings/SettingsViewController.swift]: Made them fileprivate Thanks!

**Review by kylehickinson — APPROVED** [review]

**Review by jhreis — APPROVED** [review]

**Review by jhreis — CHANGES_REQUESTED** [review]
@danishjafri88 can you fix the merge conflicts on this?

**Comment by danishjafri88** [comment]
@jhreis  merge conflicts resolved

**Review by jhreis — APPROVED** [review]

**Pull request merged**